Makefile

```diff
@@ -1,5 +1,5 @@
 SDK ?= "iphonesimulator"
-DESTINATION ?= "platform=iOS Simulator,name=iPhone 7"
+DESTINATION ?= "platform=iOS Simulator,name=iPhone 9"
```

Comment:
I don't think iPhone 9 is a thing. if you run `xcrun simctl list` you can get a list of devices and names. Maybe choose iPhone X if you want to bump the simulator to a newer one. The OS and the simulator don't always align.

Reply:
awesome thank you!